src/PalindromeQuality.cpp

```diff
@@ -12,6 +11,7 @@ using std::pow;
 #include <boost/accumulators/statistics.hpp>
 
 using namespace boost::accumulators;
+using namespace shasta;
 
 typedef accumulator_set<float, features<tag::count, tag::mean, tag::variance>> stats_accumulator;
```

Comment:
I no longer use `typedef`. Instead of `typedef A B`, use `using B = A`. It is clearer and more consistent with most of Shasta code.